tests/conftest.py

```diff
@@ -1,29 +1,36 @@
+from collections import namedtuple
+from datetime import datetime
+
+from pkg_resources import resource_filename
```

Comment:
Apparently the [use of `pkg_resources` is deprecated](https://setuptools.pypa.io/en/latest/pkg_resources.html). You may want to use the new `importlib` API if possible.